src/components/ProfileUpdate/index.jsx

```diff
@@ -62,7 +62,7 @@ const ProfileUpdate = props => {
       });
   };
   useEffect(() => {
-    let keys = ['firstName', 'lastName', 'major', 'bio'];
+    let keys = ['firstName', 'lastName', 'major', 'bio', 'graduationYear'];
```

Comment:
Is there a reason why this is a let vs. a const?

Reply:
Where is the prop type for grad year 
![image](https://user-images.githubusercontent.com/8893880/81467035-d13a6180-918a-11ea-9838-0460b21eaabf.png)